test/functional/admin_controller_test.rb

```diff
@@ -388,4 +388,8 @@ def teardown
     assert_equal 1, node.author.status
     assert_redirected_to node.path(:question)
   end
+
+  test 'fail a test' do
```

Comment:
Just to keep this as a reminder we should remove this test once we have completed this